### ISSUE TITLE
Node xception_65 incompatible shapes when training deeplab on GPU

### ISSUE THREAD

**Issue by nat347** [tensorflow/models#6778]
### System information
- **What is the top-level directory of the model you are using**: deeplab
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Debian GNU/Linux 9.9
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.13.1
- **Bazel version (if compiling from source)**: -
- **CUDA/cuDNN version**: 10.0/7.4
- **GPU model and memory**: Tesla V100/16GB
- **Exact command to reproduce**: Run training script on GPU

### Describe the problem
When running the train.py script of deeplab on a custom dataset using GPU I'm getting incompatible shape messages regarding parameters of the xception_65 backbone model. Interestingly, the numbers are correct but translated one step to the right.

Also, when I run the code on CPU by setting
os.environ["CUDA_VISIBLE_DEVICES"]="-1" in train.py,
the model is able to train and no incompatible shape issues appear.

Has anyone encountered a similar issue and/or knows how to solve it in order to make the script run on GPU? Thank you!

### Source code / logs
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Saving checkpoints for 0 into .../model.ckpt.
Total loss is :[2.90530109]
2019-05-14 15:16:19.214232: I tensorflow/stream_executor/dso_loader.cc:152] successfully opened CUDA library libcupti.so.10.0 locally
WARNING:tensorflow:From .../anaconda3/lib/python3.6/site-packages/tensorflow/python/profiler/internal/flops_registry.py:142: tensor_shape_from_node_def_name (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.compat.v1.graph_util.remove_training_nodes
411 ops no flops stats due to incomplete shapes.
Parsing Inputs...
Node xception_65/entry_flow/conv1_1/Conv2D incompatible shapes: Shapes (?, 240, 320, 32) and (4, 32, 240, 320) are not compatible.
Node xception_65/entry_flow/conv1_1/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 240, 320, 32) and (4, 32, 240, 320) are not compatible.
Node xception_65/entry_flow/conv1_1/Relu incompatible shapes: Shapes (?, 240, 320, 32) and (4, 32, 240, 320) are not compatible.
Node xception_65/entry_flow/conv1_2/Conv2D incompatible shapes: Shapes (?, 240, 320, 64) and (4, 64, 240, 320) are not compatible.
Node xception_65/entry_flow/conv1_2/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 240, 320, 64) and (4, 64, 240, 320) are not compatible.
Node xception_65/entry_flow/conv1_2/Relu incompatible shapes: Shapes (?, 240, 320, 64) and (4, 64, 240, 320) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/shortcut/Conv2D incompatible shapes: Shapes (?, 120, 160, 128) and (4, 128, 120, 160) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/Relu incompatible shapes: Shapes (?, 240, 320, 64) and (4, 64, 240, 320) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/shortcut/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 120, 160, 128) and (4, 128, 120, 160) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/separable_conv1_depthwise/depthwise incompatible shapes: Shapes (?, 240, 320, 64) and (4, 64, 240, 320) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/separable_conv1_depthwise/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 240, 320, 64) and (4, 64, 240, 320) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/separable_conv1_pointwise/Conv2D incompatible shapes: Shapes (?, 240, 320, 128) and (4, 128, 240, 320) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/separable_conv1_pointwise/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 240, 320, 128) and (4, 128, 240, 320) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/Relu_1 incompatible shapes: Shapes (?, 240, 320, 128) and (4, 128, 240, 320) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/separable_conv2_depthwise/depthwise incompatible shapes: Shapes (?, 240, 320, 128) and (4, 128, 240, 320) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/separable_conv2_depthwise/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 240, 320, 128) and (4, 128, 240, 320) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/separable_conv2_pointwise/Conv2D incompatible shapes: Shapes (?, 240, 320, 128) and (4, 128, 240, 320) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/separable_conv2_pointwise/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 240, 320, 128) and (4, 128, 240, 320) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/Relu_2 incompatible shapes: Shapes (?, 240, 320, 128) and (4, 128, 240, 320) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/Pad incompatible shapes: Shapes (?, 242, 322, 128) and (4, 128, 242, 322) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/separable_conv3_depthwise/depthwise incompatible shapes: Shapes (?, 120, 160, 128) and (4, 128, 120, 160) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/separable_conv3_depthwise/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 120, 160, 128) and (4, 128, 120, 160) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/separable_conv3_pointwise/Conv2D incompatible shapes: Shapes (?, 120, 160, 128) and (4, 128, 120, 160) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/separable_conv3_pointwise/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 120, 160, 128) and (4, 128, 120, 160) are not compatible.
Node xception_65/entry_flow/block1/unit_1/xception_module/add incompatible shapes: Shapes (?, 120, 160, 128) and (4, 128, 120, 160) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/shortcut/Conv2D incompatible shapes: Shapes (?, 60, 80, 256) and (4, 256, 60, 80) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/Relu incompatible shapes: Shapes (?, 120, 160, 128) and (4, 128, 120, 160) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/shortcut/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 60, 80, 256) and (4, 256, 60, 80) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/separable_conv1_depthwise/depthwise incompatible shapes: Shapes (?, 120, 160, 128) and (4, 128, 120, 160) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/separable_conv1_depthwise/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 120, 160, 128) and (4, 128, 120, 160) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/separable_conv1_pointwise/Conv2D incompatible shapes: Shapes (?, 120, 160, 256) and (4, 256, 120, 160) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/separable_conv1_pointwise/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 120, 160, 256) and (4, 256, 120, 160) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/Relu_1 incompatible shapes: Shapes (?, 120, 160, 256) and (4, 256, 120, 160) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/separable_conv2_depthwise/depthwise incompatible shapes: Shapes (?, 120, 160, 256) and (4, 256, 120, 160) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/separable_conv2_depthwise/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 120, 160, 256) and (4, 256, 120, 160) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/separable_conv2_pointwise/Conv2D incompatible shapes: Shapes (?, 120, 160, 256) and (4, 256, 120, 160) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/separable_conv2_pointwise/BatchNorm/FusedBatchNorm incompatible shapes: Shapes (?, 120, 160, 256) and (4, 256, 120, 160) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/Relu_2 incompatible shapes: Shapes (?, 120, 160, 256) and (4, 256, 120, 160) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/Pad incompatible shapes: Shapes (?, 122, 162, 256) and (4, 256, 122, 162) are not compatible.
Node xception_65/entry_flow/block2/unit_1/xception_module/separable_conv3_depthwise/depthwise incompatible shapes: Shapes (?, 60, 80, 256) and (4, 256, 60, 80) are not compatible.
...

**Comment by Jasonfinish** [comment]
Yeah, I stuck with the some issue, but I do not know what result in ? I need help too

**Comment by powea000** [comment]
I got the same issue when use auto-deeplab.

**Comment by whjpython** [comment]
me too，How to solve this problem？

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
